app/controllers/shopping_cart_controller.rb

```diff
@@ -20,6 +20,19 @@ def add_product_to_cart
     redirect_to cart_path
   end
 
+  def destroy_cart_items
+    cart_items = current_user.shopping_cart.cart_items
+    if cart_items.any?
+      cart_items.destroy_all
+      flash[:notice] = 'Your shopping cart has been cleared'
+    elsif cart_items.empty?
+      flash[:alert] = 'Your shopping cart is empty and cannot be cleared'
+    else
```

Comment:
W jakiej sytuacji to może wystąpić?
Jeśli mamy kartę wystąpi sytuacja 1, jeśli nie 2? 3 w przypadku braku shopping carty? ale to już wyżej powinno nam rzucić errorem